clients/src/test/java/org/apache/kafka/clients/consumer/internals/FetcherTest.java

```diff
@@ -1332,6 +1333,24 @@ public void testGetOffsetsForTimes() {
         testGetOffsetsForTimesWithError(Errors.BROKER_NOT_AVAILABLE, Errors.NONE, 10L, 100L, 10L, 100L);
     }
 
+    @Test(expected = TimeoutException.class)
+    public void testBatchedListOffsetsMetadataErrors() {
+        client.reset();
```

Comment:
Hmm, it's not clear to me why we need `reset` here and in `setup`. The test still passes with them?